src/main/java/frc/robot/RobotContainer.java

```diff
@@ -59,8 +64,16 @@ public class RobotContainer {
     private LEDs leds = new LEDs();
     private final Swerve s_Swerve;
     private final Vision s_Vision;
+    private CoralScoring coralScoring;
     private Climber climb;
 
+    /* Triggers */
+
+    private Trigger intakedCoralRight =
+        new Trigger(() -> coralScoring.getGrabingRightBeamBrakeStatus());
+
+    private Trigger outtakedCoral = new Trigger(() -> coralScoring.getScoringBeamBrakeStatus());
+
```

Comment:
These should move to the Subsystem since they are self contained and don't use properties from multiple subsystems